docgen/src/examples/e-commerce-infinite/App.js

```diff
@@ -204,21 +204,13 @@ const Hit = ({ item }) => {
   );
 };
 
-const CustomResults = createConnector({
-  displayName: 'CustomResults',
-
-  getProvidedProps(props, searchState, searchResults) {
-    const noResults = searchResults.results
-      ? searchResults.results.nbHits === 0
-      : false;
-    return { query: searchState.query, noResults };
-  },
-})(({ noResults, query }) => {
-  if (noResults) {
+const CustomResults = connectStateResults(({ searchState, searchResult }) => {
+  if (searchResult && searchResult.nbHits === 0) {
```

Comment:
here I think you'd also have advantage of the destructuring as described in #405